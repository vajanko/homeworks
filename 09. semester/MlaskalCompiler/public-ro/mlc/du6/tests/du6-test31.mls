program du6test;

label 1,2,3,4,5,6;

begin
  goto 7;
end.