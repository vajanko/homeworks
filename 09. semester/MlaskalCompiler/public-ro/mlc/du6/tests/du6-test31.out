Error (line 6): '7' is not a label
Warning (line 3): Label '1' declared but never used
Warning (line 3): Label '2' declared but never used
Warning (line 3): Label '3' declared but never used
Warning (line 3): Label '4' declared but never used
Warning (line 3): Label '5' declared but never used
Warning (line 3): Label '6' declared but never used
There were some errors 1 and warnings 6
